### PR TITLE
Add "trojanx" v0.4

### PR DESCRIPTION
**Pull request by singularitti**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew install --cask {{cask_file}}` worked successfully.
- [x] `brew uninstall --cask {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by singularitti** [comment]
Could this be merged? 😅

**Comment by miccal** [comment]
Thank you @singularitti.

**Review by miccal — APPROVED** [review]

**Comment by LRNZ09** [comment]
There's a typo, `ununstall`

File: Casks/trojanx.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  appcast "https://github.com/JimLee1996/TrojanX/releases.atom"
+  name "TrojanX"
+  desc "Mechanism to bypass the Great Firewall"
+  homepage "https://github.com/JimLee1996/TrojanX"
+
+  depends_on macos: ">= :el_capitan"
+
+  app "TrojanX.app"
+
+  ununstall delete: "/Library/Application Support/TrojanX"
```
Comment (LRNZ09) [inline Casks/trojanx.rb:15]: Typo
```suggestion
  uninstall delete: "/Library/Application Support/TrojanX"
```

**Comment by miccal** [comment]
Thanks @LRNZ09 -- fixed in https://github.com/Homebrew/homebrew-cask/pull/95824.

**Pull request merged**